python/tests/test_tables.py

```diff
@@ -2361,6 +2361,20 @@ def test_samples_interface(self):
             with pytest.raises(OverflowError):
                 tables.simplify(samples=np.array([0, bad_node]))
 
+    def test_bad_individuals(self, simple_ts_fixture):
+        tables = simple_ts_fixture.dump_tables()
+        tables.individuals.clear()
+        tables.individuals.add_row(parents=[-2])
+        with pytest.raises(tskit.LibraryError, match="Individual out of bounds"):
+            tables.simplify()
+
+    def test_unsorted_individuals(self, simple_ts_fixture):
+        tables = simple_ts_fixture.dump_tables()
+        tables.individuals.clear()
+        tables.individuals.add_row(parents=[1])
+        tables.individuals.add_row(parents=[-1])
+        tables.simplify()
```

Comment:
does this test anything?

Reply:
No, I don't think so.

We should check that the integrity of individual refs is maintained. A good test here could be to take a forward sim sim with parent references, shuffle them, simplify, and then use the metadata to confirm that the correct individuals are parents at the end of simplify.

Reply:
This test belongs in #1186, I'll add it there.